adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/AdeleStyle.java

```diff
@@ -0,0 +1,32 @@
+package com.microservicesteam.adele.ticketmaster;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+import org.immutables.value.Value.Style.BuilderVisibility;
+import org.immutables.value.Value.Style.ImplementationVisibility;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.PACKAGE;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.CLASS;
+
+@Target({PACKAGE, TYPE})
+@Retention(CLASS)
+@JsonSerialize
+@JsonDeserialize
+@Value.Style(
+    get = {"*"},
+    init = "*",
```

Comment:
@kelemena I copied `AdeleStyle` here too, since I needed the JSON related settings. I know that these modules will live their own life as services later, but should we have a standard agreement on these style files across different modules/services?